CHANGELOG.md

```diff
@@ -210,6 +210,13 @@
 
 🔧 Fixes:
 
+- Fix improperly indented html output in Design System examples
+
+  Thanks to [Debs](https://github.com/debsdee) and [Joe](https://github.com/joelanman) for raising this issue.
+
+([PR #1353](https://github.com/alphagov/govuk-frontend/pull/1353))
```

Comment:
Minor, but this should be indented so it sits with the rest of the bullet point. (Sorry, didn't notice this first time)

Reply:
No worries, updated.